### PR TITLE
2.1: Remove submodules for repos that merged to aspnet/Extensions

### PR DESCRIPTION
**Pull request by natemcmaster**
React to https://github.com/aspnet/Extensions/pull/419 -- aspnet/Extensions implements the ProdCon contract now. Builds of the Extensions repo go through ProdCon instead of building as a submodule of this repo.

### CODE REVIEW

**Review by Eilon — APPROVED** [review]

**Comment by natemcmaster** [comment]
⌚️ waiting until ProdCon changes are ready to merge.

**Pull request merged**
